### DISCUSSION TITLE
Future Feature : Fase 2 Rawat Jalan - Data Object Class

### DISCUSSION THREAD

**Discussion by ivanwilliammd** [ivanwilliammd/satusehat-integration#126] [Ideas]
**Is your feature request related to a problem? Please describe.**
Future future Rawat Jalan Fase 2 : 
- Observation tanda-tanda vital based on LOINC
- Procedure based on ICD 9 CM
- Composition (education) based on LOINC 

**Describe the solution you'd like**
Membuat FHIR class resource Observation, Procedure, Composition

**Describe alternatives you've considered**
Updating 3 class baru tersebut pada src/FHIR

**Additional context**
https://satusehat.kemkes.go.id/platform/docs/id/api-catalogue/integrations/apis/procedure/
https://satusehat.kemkes.go.id/platform/docs/id/api-catalogue/integrations/apis/composition/
https://satusehat.kemkes.go.id/platform/docs/id/api-catalogue/integrations/apis/observation/

**Comment by afifsyaifullah1512** [comment]
Malam dok, apakah belum ada update untuk observation terkhusus untuk tipe bundle?

**Comment by ivanwilliammd** [comment]
Sementara ini belum ada volunteer, apakah @afifsyaifullah1512 tertarik untuk contribute?
Bila iya, bisa infokan saya untuk saya create branch untuk commit PR-nya

**Comment by 4RSIM3R** [comment]
Halo pak dokter, saya berminat untuk volunteer di project ini, seems untuk seeder wilayah blm ter publish juga di service provider nya, for more detail, izin meng-kontak pak dokter yah

**Comment by ivanwilliammd** [comment]
Selamat malam @4RSIM3R 

Saat ini belum ada konvensi aturan coding yang spesifik Mas, sementara mengikuti standar PSR-4 autoload composer saja.

Terkait penulisan kodifikasi library menggunakan fungsi native PHP yang backward compatible dari 7.4 s/d 8.x mengingat pengguna library ini tidak semuanya sdh pakai PHP8.

Untuk branching rilis mengikuti format yang tersedia berupa 1.1.1, akan tetapi kalau untuk branching PR --> bisa buat issue-nya dahulu lalu create PR

Terimakasih.🙏🏻
